src/pages/Timetable/components/MyLectureTimetable/TotalGrades.tsx

```diff
@@ -0,0 +1,19 @@
+import { LectureInfo, TimetableLectureInfo } from 'interfaces/Lecture';
+import styles from './MyLectureTimetable.module.scss';
+
+function TotalGrades(props: { myLectureList: TimetableLectureInfo[] | LectureInfo[] }) {
```

Comment:
```suggestion
function TotalGrades(myLectureList: TimetableLectureInfo[] | LectureInfo[]) {
```
props없이 쓰는 것이 훨씬 깔끔해보여요 `props.myLectureList`와 같이 굳이 props를 쓰기보다 바로 `myLectureList`에 접근하면 훨씬 편할 것 같아요!
이렇게하면 아래 린트에러를 disable하는 문구도 삭제할 수 있어요!

Reply:
반영했습니다! c7aea3eff6b7c42518a0855c5f12503436fd585c